src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1684,4 +1684,22 @@
   <data name="CryptoError_Sha1" xml:space="preserve">
     <value>Unexpected cryptographic error while computing SHA1 ({0}).</value>
   </data>
+  <data name="ClassLoad_General" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}'.</value>
+  </data>
+  <data name="ClassLoad_MissingMethodRva" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}' because the method '{2}' has no implementation (no RVA).</value>
```

Comment:
We'll need to add some sort of `ValidateType` step that will be called whenever codegen touches a type anyway so my plan was to eventually move it there.

(We'll need to eagerly check that we can e.g. generate EEType because if we find out that one of it's base types is missing at the time when we already generated code that uses it, we'll have no other option than to fail compilation. If we validate beforehand, we can just replace the body that referenced it with a throw and move on.)